openapi/components/schemas/Customer.yaml

```
@@ -100,8 +100,8 @@ properties:
     default: false
   hasFulfilledKyc:
     description: >-
-      Whether the customer has fulfilled a KYC request by providing valid
-      identity and address documents.
+      Whether the customer has valid identity and address documents
```

nitpick: I think we should avoid starting descriptions with "whether"

```suggestion
      Specifies whether the customer provided valid identity and address documents
```